### ISSUE TITLE
Input redundancy computed incorrectly for k=1

### ISSUE THREAD

**Issue by austin-marcus** [rionbr/CANA#28]
This [section](https://github.com/rionbr/CANA/blob/85d9df7178073f0b9fd3018c936c250ae7496f6d/cana/boolean_node.py?plain=1#L155) causes the input redundancy of a single-variable constant function to be erroneously calculated as 0, when it should be 1.

Example:
```
from cana.boolean_node import BooleanNode as BN

print(BN(k=1, outputs=[0,0]).input_redundancy())
print(BN(k=1, outputs=[1,1]).input_redundancy())
print(BN(k=1, outputs=[0,1]).input_redundancy())
```
produces:

```
0.0
0.0
0.0
```

and I believe it should be:

```
1.0
1.0
0.0
```

**Comment by austin-marcus** [comment]
I believe [Felipe's fork](https://github.com/fcphysics/CANA) already has a fix for this.

**Comment by austin-marcus** [comment]
Resolved by #30.
